server/portal/apps/projects/management/commands/migrate-projects.py

```diff
@@ -46,6 +47,8 @@ def handle(self, *args, **options):
                     project_id=meta.project_id
                 ))
 
+                index_project.apply_async(args=[meta.project_id])
```

Comment:
Indexing will only occur for projects who are missing their PI: 

```metadatas = ProjectMetadata.objects.all().filter(pi=None)
        for meta in metadatas:
```

You'll want to index all projects, right?

----
Should the help text for this command should be updated to reflect the indexing.

Reply:
Good catch!